apps/modernization-ui/src/apps/page-builder/pages/PageBuilder/PageBuilder.tsx

```diff
@@ -2,6 +2,7 @@ import { PageBuilderSideNav } from '../../components/Navigation/PageBuilderSideN
 import './PageBuilder.scss';
 
 type Props = {
+    page?: string;
```

Comment:
Question: Is this needed? the `PageBuilder.tsx` is no longer used in the new `Edit` page so there should be no need to make changes to it.